src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1570,10 +1565,8 @@ public String getFeedbackSessionResultSummaryInSectionAsCsv(String courseId,
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, feedbackSessionName);
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, section);
 
-        return feedbackSessionsLogic.getFeedbackSessionResultsSummaryInSectionAsCsv(feedbackSessionName,
-                                                                                    courseId,
-                                                                                    instructorEmail,
-                                                                                    section);
+        return feedbackSessionsLogic.getFeedbackSessionResultsSummaryInSectionAsCsv(
+                feedbackSessionName, courseId, instructorEmail, section);
```

Comment:
Although 8 spaces works, maybe it's neater to indent 4 spaces from `[f]eedbackSessionsLogic...`?